src/style.css

```diff
@@ -0,0 +1,134 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__content-container {
+  display: flex;
+  flex-direction: column;
+  margin: 36px 16px 16px;
+}
+
+.card__logo {
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  color: #060b35;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+
+  margin: 0;
+}
+
+.card__code {
+  font-size: 10px;
+  color: #616070;
+  line-height: 14px;
+  font-weight: 400;
+
+  margin: 4px 0 0;
+}
+
+.stars {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__container {
+  display: flex;
+}
+
+.stars__image {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-size: 13px 12px;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+}
+
+.stars__image:last-child {
+  background-image: url(./images/star.svg);
+}
+
+.stars__image:not(:last-child) {
```

Comment:
[checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 11th item 4th item, you should have all the modifiers selectors, from 1 to 5 stars